bust-a-move-framework/src/test/java/bustamove/game/CannonTest.java

```diff
@@ -43,8 +44,9 @@ public void fireMaxRighttAngle() {
         for (int i = 0; i < 60; i++) {
             cannon.stepUp();
         }
+        Bubble loaded = game.getBubbles().get(1);
```

Comment:
Not that. The order of bubbles changes in a weird manner when one gets shot.
Cannon instantiated
CurrB -> NextB
fire
CurrB -> FiredBubble -> NextB
When a bubble gets shot, the shot bubble moves to the second position and the NextBubble becomes the current one, which is located on the first position. They are kinda swapping places. I myself would prefer to keep the bubbles in a fixed order, but other tasks currently have higher priority.